lib/browser/api/ipc-main.ts

```diff
@@ -1,8 +1,40 @@
 import { EventEmitter } from 'events'
+import { IpcMainInvokeEvent } from 'electron'
 
-const emitter = new EventEmitter()
+class IpcMain extends EventEmitter {
```

Comment:
We can set this up so that you don't have to type `handle` and `handleOnce` and such.  Also ensuring type safety / matching of our docs.

```
class IpcMain extends EventEmitter implements Electron.IpcMain {
    handle: Electron.IpcMain['handle'] = (method, fn) => {

    }

    handleOnce: Electron.IpcMain['handleOnce'] = (method, fn) => {

    }
}
```

In that example both `method` and `fn` have types from the docs rather than re-declaring